library/src/main/java/com/mapzen/android/MapzenMap.java

```diff
@@ -394,4 +395,129 @@ public boolean isSimultaneousGestureAllowed(TouchInput.Gestures first,
             TouchInput.Gestures second) {
         return mapController.isSimultaneousGestureAllowed(first, second);
     }
+
+    /**
+     * Draws two pins on the map. The start pin is active and the end pin is inactive.
+     * @param start
+     * @param end
+     */
+    public void drawRoutePins(LngLat start, LngLat end) {
+        overlayManager.drawRoutePins(start, end);
+    }
+
+    /**
+     * Clears the start and end pins from the map.
+     */
+    public void clearRoutePins() {
+        overlayManager.clearRoutePins();
+    }
+
+    /**
+     * Draws a dropped pin on the map at the point supplied.
+     * @param point
+     */
+    public void drawDroppedPin(LngLat point) {
+        overlayManager.drawDroppedPin(point);
+    }
+
+    /**
+     * Clears the dropped pin from the map.
+     */
+    public void clearDroppedPin() {
```

Comment:
Should this method be renamed to `clearDroppedPins()` since it clears all dropped pins from the map?